### PR TITLE
Add GitHub action to update DigitalOcean deployment

### PR DESCRIPTION
**Pull request by Vierlip**
- Add GitHub Action workflow to update DigitalOcean deployments on push to the `main` branch.
- Update and create deployments for `imgproxy`, `cms`, and `site-main` apps using `.digitalocean/deploy.sh` script.

### CODE REVIEW

File: .github/workflows/update-digital-ocean-deployment.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      - main
+jobs:
+  update-deployment:
+    name: Update DigitalOcean Deployment
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # Needed to also fetch next branch
```
Comment (dkarnutsch) [inline .github/workflows/update-digital-ocean-deployment.yml:14]:
```suggestion
```

we don't use the next branch here

Comment (Vierlip) [reply .github/workflows/update-digital-ocean-deployment.yml]: 73b19e9e076c3b34d8e22e78cdde1da8f990d770

**Review by johnnyomair — COMMENTED** [review]
Why can't we use the DigitalOcean action for this?
https://docs.digitalocean.com/products/app-platform/how-to/deploy-from-github-actions/

**Comment by Vierlip** [comment]
> Why can't we use the DigitalOcean action for this? https://docs.digitalocean.com/products/app-platform/how-to/deploy-from-github-actions/

36bb5396a0dc7a2f8a7c0963742232e43bbe8691

**Review by johnnyomair — APPROVED** [review]

**Review by dkarnutsch — APPROVED** [review]

**Pull request merged**